src/Workspaces/Core/Portable/SolutionCrawler/SolutionCrawlerOptions.cs

```diff
@@ -12,10 +12,17 @@ internal static class SolutionCrawlerOptions
         /// <summary>
         /// Option to turn configure background analysis scope for the current user.
         /// </summary>
-        public static readonly PerLanguageOption2<BackgroundAnalysisScope> BackgroundAnalysisScopeOption = new(
-            nameof(SolutionCrawlerOptions), nameof(BackgroundAnalysisScopeOption), defaultValue: BackgroundAnalysisScope.Default,
+        public static readonly PerLanguageOption2<BackgroundAnalysisScope?> BackgroundAnalysisScopeOption = new(
+            nameof(SolutionCrawlerOptions), nameof(BackgroundAnalysisScopeOption), defaultValue: null,
             storageLocation: new RoamingProfileStorageLocation($"TextEditor.%LANGUAGE%.Specific.BackgroundAnalysisScopeOption"));
 
+        /// <summary>
+        /// Feature flag to control the default background analysis scope for the current user.
+        /// </summary>
+        public static readonly Option2<bool> ActiveFileAsDefaultBackgroundAnalysisScopeFeatureFlag = new(
+            nameof(SolutionCrawlerOptions), nameof(ActiveFileAsDefaultBackgroundAnalysisScopeFeatureFlag), defaultValue: false,
+            storageLocation: new FeatureFlagStorageLocation("Roslyn.ActiveFileAsDefaultBackgroundAnalysisScopeFeatureFlag"));
```

Comment:
instead of having this be a boolean, we can have this be an `int` right?  and that way our A/B test could test *any* of the other values.  then it would just be BackgroundAnalysisScopeOverrideOption

Reply:
This code is now removed - after talking @vatsalyaagrawal, we have decided to make ActiveFile as the default without any experiment/feature flag (analyzers run just on active file, compiler warnings/errors show for all open files). Given this change is going very early in 17.2, it should give us enough dogfooding time to make any follow-up changes as required.